pdns/reczones.cc

```diff
@@ -46,6 +46,7 @@ bool primeHints(void)
 {
   // prime root cache
   const vState validationState = vState::Insecure;
+  static const ComboAddress from("255.255.255.255");
```

Comment:
No use in making this one `static`, there is no extra value to save this one between calls.